packages/strapi-plugin-content-manager/admin/src/components/Wysiwyg/MediaLib.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { useStrapi, prefixFileUrlWithBackendUrl } from 'strapi-helper-plugin';
+import PropTypes from 'prop-types';
+
+const MediaLib = ({ isOpen, onChange, onToggle }) => {
+  const {
+    strapi: {
+      componentApi: { getComponent },
+    },
+  } = useStrapi();
+  const [data, setData] = useState(null);
+  const [isDisplayed, setIsDisplayed] = useState(false);
+
+  useEffect(() => {
+    if (isOpen) {
+      setIsDisplayed(true);
+    }
+  }, [isOpen]);
+
+  const Component = getComponent('media-library').Component;
+
+  const handleInputChange = data => {
+    if (data) {
+      const { name, alternativeText, url } = data;
+      const alt = alternativeText === '' ? name : alternativeText;
```

Comment:
As `''` is false in JS, you just need to do `const alt = alternativeText || name;`